src/UI/Component/Panel/Listing/Divider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/* Copyright (c) 2017 Alex Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Listing divider
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Divider extends \ILIAS\UI\Component\Component {
```

Comment:
I just read @alex40724 other comment, sry. I somehow can see why we want to have a divider for a dropdown, but I do not like it for for the item listings. We clearly have groups there. This is the way we talk about these things and I also think we have cases where some actions over the groups are attached to its heading. That is, the group clearly spans over some items, which is a meaning that is lost when using divider.